OracleDatabase/RAC/OracleRealApplicationClusters/dockerfiles/19.3.0/Dockerfile

```diff
@@ -1,31 +1,49 @@
 # LICENSE UPL 1.0
 #
-# Copyright (c) 1982-2018 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Please remove `All rights reserved.` from your copyright lines whenever you edit them. Oracle no longer uses this phrase.